Header header
float64[] time_array
float64[] slip_array
